layouts/jobs/single.html

```diff
@@ -23,9 +23,7 @@
         </div>
       </div>
       <div class="col-span-3 bg-card-light text-white divide-y divide-page px-5 sticky top-8">
-        <div class="card-title pt-6 pb-5">Table of Contents</div>
-        <div class="nested-link font-bold py-2">{{ .TableOfContents }}</div>
-        <div>&nbsp;</div>
+        {{ partial "components/toc.html" . }}
```

Comment:
We might need to revisit this since we might have alternative toc partial(s) for the changes, faq, and bis templates.

Reply:
The FAQ ToC in particular I have no idea how we should handle.

For the rest of them, is there a conventional way to map different partials to their corresponding layout? I'm fine with breaking just it out into separate files (maybe `partials/components/toc/changes_toc.html`, `.../bis_toc.html`, etc.) if the need arises.

Reply:
I think we'll just need perhaps a `components/toc.html` and a `components/custom_toc.htm`? I think the onus of formatting stuff into a slice/dict/whatever should be on the layouts/partials that call them.